### PR TITLE
denum.d: remove unused imports

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: compiler/src/dmd/denum.d [hunk 1]
```diff
@@ -66,6 +63,8 @@ extern (C++) final class EnumDeclaration : ScopeDsymbol
     import dmd.common.bitfields : generateBitFields;
     mixin(generateBitFields!(BitFields, ubyte));
 
+    Symbol* sinit;
+
```
Comment (thewilsonator) [inline compiler/src/dmd/denum.d:67]: unrelated change

**Review by thewilsonator — APPROVED** [review]
otherwise looks good

File: compiler/src/dmd/denum.d (hunk 1 above)
Comment (WalterBright) [reply compiler/src/dmd/denum.d:67]: Just grouped it with the other field declarations. The layout hasn't changed.

Comment (dkorpel) [reply compiler/src/dmd/denum.d:67]: frontend.h needed to be updated for this change. This PR was merged with failing CI.
